### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by AdamEternal** [scottTomaszewski/obsidian-inline-admonitions#16]
Issue: This plugin does not support my device. (By extension, I assume it does not support _any_ mobile device.) Obsidian does not allow me to install or enable it on mobie.

Versions:

- Android 14
- Obsidian 1.7.6
- Inline Admonitions: 1.4.6

I tried to install Inline Admonitions via mobile, and the app didn't allow this. I then installed it on Desktop and sync'd it to the mobile copy of my vault via Obsidian Sync. It would not enable. (This is all expected behaviour, but I figured more details were better.)

**Comment by scottTomaszewski** [comment]
Yes, i need to test/enable mobile support.  I have a few things in this plugin to fix.  Finding time has been rough.  I might be able to get to knocking out a few of these this/next week!

**Comment by razvanantonescu** [comment]
Looking forward for the fix. I'm glad I wasn't the only one with the issue

**Comment by scottTomaszewski** [comment]
Version 1.6.0 has been released and supports mobile. I am only able to test on android devices - if anyone has an apple device, I would be grateful if you would let me know if it works 😅 

Thanks!
